backbone.virtual-collection.js

```diff
@@ -62,6 +53,30 @@ var VirtualCollection = Backbone.VirtualCollection = Backbone.Collection.extend(
     if (this.comparator) this.sort({silent: true});
   },
 
+  _handleParentEvent: function (event, target) {
+    var args_with_name = _.toArray(arguments);
+    var args = args_with_name.slice(1);
+    if (target == this.collection) {
+      switch (event) {
+        case 'reset':
+          return this._onReset.apply(this, args);
+        case 'sort':
+          return this._onSort.apply(this, args);
+      }
+      return this.trigger.apply(this, args_with_name);
+    } else if (target instanceof Backbone.Model) {
+      switch (event) {
+        case 'add':
+          return this._onAdd.apply(this, args);
+        case 'change':
+          return this._onChange.apply(this, args);
+        case 'remove':
+          return this._onRemove.apply(this, args);
+      }
+      return this.trigger.apply(this, args_with_name);
```

Comment:
we didn't use to proxy all events on the parent collection. is this really desired